addons/controls/README.md

```diff
@@ -84,23 +84,39 @@ export const Reflow = () => {
   const label = text('Label', 'reflow');
   return (
     <>
-      {Array.from({ length: count }, (_, i) => (
-        <Button label={`button ${i}`} />
+      {[...Array(count)].map((_, i) => (
+        <Button key={i} label={`button ${i}`} />
       ))}
     </>
   );
 };
-```
+``` 
 
 And again, as above, this can be rewritten using [fully custom args](https://storybook.js.org/docs/react/essentials/controls#fully-custom-args):
 
 ```jsx
 export const Reflow = ({ count, label, ...args }) => (
+  <>
+    {[...Array(count)].map((_, i) => (
+        <Button key={i} label={`${label} ${i}`} {...args} />
+    ))}
+  </>
   <>{Array.from({ length: count }, (_, i) => <Button label={`${label} ${i}` {...args}} />)}</>
```

Comment:
Doesn't the previous line make this line unnecessary?

Reply:
Yes it should have been deleted, I deleted it and undo it accidentally, will re-delete now :)

Reply:
*line removed*